deep_trekker.orogen

```diff
@@ -28,6 +28,9 @@ task_context "RevolutionTask" do
     property "powered_reel_update_interval", "base/Time"
     # How often should a request be sent to update the revolution pose z attitude
     property "pose_z_attitude_update_interval", "base/Time"
+    # Matrix to correct the behavior in drive commands for different axis
+    # and to add offset commands
```

Comment:
Add that it MUST be 6x7 with and try to describe the expectations for each element, something like 
[ Tx Ty Tz Rx Ry Rz OffX ] -> for X axis
[ Tx Ty Tz Rx Ry Rz OffY] -> for Y axis
and so on